de/README.md

```diff
@@ -0,0 +1,76 @@
+# Creative Commons Licenses für GitHub-Projekte
+
```

Comment:
I would add a badge for returning to the English version. Do you agree?

Reply:
Yep, also thought about it, I'll include it!